data-preprocessing/JSON_Section_Extractor/bookPaths.json

```diff
@@ -0,0 +1,3 @@
+{
+    "path_to_parsed_books": "./parsed_books/"
+}
```

Comment:
add a newline to get rid of the symbol here. it's a line ending discrepancy

Reply:
Will work on it